dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -1109,36 +1187,23 @@ export const Card = ({
 								{showPill ? (
 									<>
 										<MediaOrNewsletterPill />
-										{format.theme === ArticleSpecial.Labs &&
-											branding && (
-												<CardBranding
-													branding={branding}
-													onwardsSource={
-														onwardsSource
-													}
-													containerPalette={
-														containerPalette
-													}
-												/>
-											)}
+										{!showLabsRedesign &&
+											format.theme ===
+												ArticleSpecial.Labs &&
+											branding &&
+											getBranding()}
```

Comment:
seems like `getBranding` already checks for branding so we can get rid of the `branding &&` conditional here and other conditionals below